examples/components/PNEUMONIA/traininsilo/conda.yaml

```diff
@@ -0,0 +1,14 @@
+name: pneumonia_train_conda_env
+channels:
+  - defaults
+  - pytorch
+dependencies:
+  - python=3.7.11
+  - pip=20.0
```

Comment:
Nit: pip=20.0 seems to be quite old/outdated. Is there any particular reason to use this old version pip? Or you just got this from Harmke's example and thus didn't spend time adjusting it? 

Might be slightly better to use less outdated version, but it is a tiny issue, and I'm ok if you want to merge the PR ASAP as it is.

Reply:
You're making a good point here too. I have no reason other than "this is what we were using for MNIST traininsilo" :) 

I have created [an issue](https://github.com/Azure-Samples/azure-ml-federated-learning/issues/180) for fixing this in a subsequent PR.